src/main/xerus/monstercat/api/response/ConnectPlaylist.kt

```diff
@@ -0,0 +1,18 @@
+package xerus.monstercat.api.response
+
+import com.google.api.client.util.Key
+
+data class ConnectPlaylist(
+		@Key("_id") var id: String = "",
+		@Key var name: String = "",
+		@Key var public: Boolean = false,
+		@Key var deleted: Boolean = false,
+
+		@Key var tracks: List<Track> = arrayListOf()
+) {
+	fun init(): ConnectPlaylist {
```

Comment:
Is this really necessary?

Reply:
Ask this to your other responses as well.

Reply:
Removed in 13492d640e1453209e1644de412c43cfcd8e8792